### ISSUE TITLE
Wiki link in the README isn't working

### ISSUE THREAD

**Issue by lucamarchionni** [stack-of-tasks/tsid#246]
The link to https://github.com/stack-of-tasks/tsid/wiki is redirected to https://github.com/stack-of-tasks/tsid

**Comment by andreadelprete** [comment]
You're right, it seems the wiki has disappeared without me noticing. Maybe @Guillaume227 knows more about this?

**Comment by nim65s** [comment]
I was not aware we had a wiki 😅 

I fixed it :)

But I'm not sure how up-to-date it is.
